data/distrodefs/rhel-10/imagetypes.yaml

```diff
@@ -2373,3 +2394,87 @@ image_types:
             - xfsprogs
             - xrdb
             - xz
+
+  container: &container
+    filename: "container.tar"
+    mime_type: "application/x-tar"
+    image_func: "container"
+    bootable: false
+    exports: ["container"]
+    required_partition_sizes: *default_required_dir_sizes
```

Comment:
(nitpick) AFAICT we don't need this as we have no partition table for the container(s)